addons/mail/controllers/discuss/channel.py

```diff
@@ -57,11 +57,9 @@ def discuss_channel_messages(self, channel_id, search_term=None, before=None, af
             ("model", "=", "discuss.channel"),
             ("message_type", "!=", "user_notification"),
         ]
-        res = request.env["mail.message"]._message_fetch(
-            domain, search_term=search_term, before=before, after=after, around=around, limit=limit
-        )
+        res = request.env["mail.message"]._message_fetch(domain, **(fetch_params or {}))
         messages = res.pop("messages")
-        if not request.env.user._is_public() and not around:
+        if not request.env.user._is_public() and fetch_params and not fetch_params.get("around"):
```

Comment:
If there's no `fetch_params` it should be True (it means there is no `around`).

Reply:
I'm not really sure if we still need to bother with that condition though. Even in the case of "around" the messages are displayed, so it would make sense to mark them as done.

I would be nice to double check why we introduced that originally.